autogpts/autogpt/autogpt/config/config.py

```diff
@@ -241,6 +241,10 @@ class Config(SystemSettings, arbitrary_types_allowed=True):
     # Stable Diffusion
     sd_webui_auth: Optional[str] = UserConfigurable(from_env="SD_WEBUI_AUTH")
 
+    # White/Black Listing URLs
+    web_policy: Optional[str] = os.getenv("WEB_POLICY", 0)
+    url_list: Optional[list] = os.getenv("URL_LIST",[]).split(",")
```

Comment:
This is not how the config of the application works, as you can see from how all the other config attributes are defined.